### ISSUE TITLE
Clarifying: Resolving import at-rules not resolving webpack's resolve.modules?

### ISSUE THREAD

**Issue by chardskarth** [webpack/sass-loader#1064]
Documentation Is:
It's mentioned [here](https://github.com/webpack-contrib/sass-loader#resolving-import-at-rules) that resolution is done:
1. First, by resolving as a relative path
2. If not found, it will try to search through the `node_modules` directory.

- [x] Missing
- [ ] Needed
- [ ] Confusing
- [x] Not Sure?

### Please Explain in Detail...

I think it should be mentioned that [webpack's alias](https://webpack.js.org/configuration/resolve/#resolvealias) is tried to be resolved before the "node_module" resolution part.

Which brings me to my next question:
**Is it expected that [webpack's resolve.modules](https://webpack.js.org/configuration/resolve/#resolvemodules) is not used as an alternative for resolution?**

### Your Proposal for Changes

Indicate, under `Webpack provides an [advanced mechanism to resolve files](https://webpack.js.org/concepts/module-resolution/).` that specifying a resolution via webpack's resolve alias can be used.

**Comment by alexander-akait** [comment]
@chardskarth Yes, alias is prefered, it was simplifed to show `~` changed, resolving logic is more complex, even more you can change `node_modules` and inject own plugins to resolver

**Comment by chardskarth** [comment]
@alexander-akait is it a future improvement to support webpack's resolve.modules option?

**Comment by alexander-akait** [comment]
I mean you are right, but if we want describe all resolver's steps it will be very big and complex, anyway if you want to improve docs - feel free to send a PR
